distribution/org.wso2.developerstudio.eclipse.distribution.project/src/org/wso2/developerstudio/eclipse/distribution/project/ui/wizard/ProjectImportWizard.java

```diff
@@ -85,4 +91,17 @@ private void deleteGraphicalSynapseConfigDir(IProject project) throws CoreExcept
 		}
 	}
 	
+	private void deleteGraphicalSynapseConfigFiles(IProject project) throws CoreException {
+		IResource[] projectResources = project.members();
+		if (projectResources != null) {
```

Comment:
It is not needed. If there is no resource in the project it will throw coreException. I can be removed.